model-optimizer/extensions/middle/InsertLayoutPropagationTransposes.py

```diff
@@ -80,14 +80,11 @@ def find_and_replace_pattern(self, graph: Graph):
                 reinterp_shape_node_id, graph.dump_graph_for_graphviz())
             input_shape = reinterp_shape_node.in_node(0).shape
             if self.is_nchw_to_nhwc_transpose_needed(reinterp_shape_node):
-                order_const = Const(graph, {'value': PermuteAttrs().get_nchw_to_nhwc_permutation(len(input_shape)).perm
-                                            }).create_node()
-                permute_node = Transpose(graph,
-                                         {'name': reinterp_shape_node.in_port(0).get_source().node.name + '/Transpose'
-                                          }).create_node()
+                permute_node = create_op_node_with_second_input(
```

Comment:
The line `order_const.infer(order_const)` has been removed. I think that it was there for some reason. Please, restore the infer function call.

Reply:
Infer function call restored